include/zephyr/drivers/fuel_gauge/max17048.h

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Since we haven't added a trigger mechanism to the fuel gauge API I think having this public header for a specific driver is fine.

However, when we do add the public trigger mechanism, we'll have to remove this.

Reply:
Completely agree with you. For the moment, this was the way I found to make such functionalities available. In the future would be great that such calls were common to all fuel-gauges (as these functionalities are quite common in such devices).